app/code/Magento/Backend/App/Response/Http/FileFactory.php

```diff
@@ -95,18 +95,20 @@ protected function _redirect($path, $arguments = [])
      * @param string $baseDir
      * @param string $contentType
      * @param int $contentLength    explicit content length, if strlen($content) isn't applicable
+     * @param string $dispositionType defines where to show content e.g. attachment or inline
      * @return \Magento\Framework\App\ResponseInterface
      */
     public function create(
         $fileName,
         $content,
         $baseDir = DirectoryList::ROOT,
         $contentType = 'application/octet-stream',
-        $contentLength = null
+        $contentLength = null,
+        $dispositionType = 'attachment'
```

Comment:
Unfortunately, we can't change public methods signature due to [Backward Compatibility](https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/)

Reply:
@sidolov should we add this change to the Magento 2.3-develop branch instead? Or isn't there a possibility to contribute new parameters yet?

Reply:
We recommend to add a new method with new parameters, deprecate the old method and instead of old implementation make a proxy call to the new method

Reply:
@sidolov Adjustments have been made, please check again. Thanks! :)